sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_02_alter/answers/_01_alter_add_hard_default_yes_tstz.answer

```diff
@@ -131,8 +131,8 @@ a    c_tstz1    c_tstz2    c_tstz3
 0
 ===================================================
```

Comment:
Shouldn't it be changed to the same content below?
```
sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_02_alter/answers/_01_alter_add_hard_default_yes_tstz.answer_cci
sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_02_alter/answers/_01_alter_add_hard_default_yes_tstz.answer_win
```

Reply:
Reflected the review. Please check again.